manual/parts/bootup.tex

```diff
@@ -118,14 +118,15 @@ \section{\label{ch:bootup:bootinfo}BootInfo Frame}
   \end{center}
 \end{table}
 
-Depending on the architecture and platform there might be additional pieces of boot
-information. If \texttt{extraLen} is greater than zero, then 4K after the start of bootinfo
-is a region of extraLen bytes containing additional bootinfo structures. Each chunk starts
-with a \texttt{seL4\_BootInfoHeader}, described in \autoref{tab:bi_header_struct}, that
-describes what the chunk is and how long it is, where the length includes the header. The
-length can be used to skip over chunks that you do not understand. The only generally
-defined chunk type is \texttt{SEL4\_BOOTINFO\_HEADER\_PADDING} and describes an empty
-chunk that has no data, any other types are platform or architecture specific. The
+The size of the fixed Boot Info Frame is \texttt{SEL4\_BI\_FRAME\_SIZE}. In the standard
+configuration, this is one page, which is 4 KiByte on x86, ARM and RISC-V. Depending on the
```

Comment:
```suggestion
configuration, this is 4 KiByte on x86, Arm and RISC-V. Depending on the
```

Reply:
Mentioning that it's a page was intentional so have an explanation for the magic 4 KiB value. We don't have any other architecture with different page sizes, where all this might get more interesting then...

Reply:
Ok, fair enough.